app/src/main/java/com/sdp13epfl2021/projmag/notification/Constants.kt

```diff
@@ -0,0 +1,10 @@
+package com.sdp13epfl2021.projmag.notification
+
+class Constants {
+    companion object{
+        const val BASE_URL = "https://fcm.googleapis.com"
+        const val SERVER_KEY= "AAAAwNSBlt8:APA91bHNLxj_wt6HCQ9Lfk3_6sSYIYHxfSRA7qgR_yZP-zCYxpgvBHlttayDPn20zKundac4-3RmyrcNwJPZagYLLuXOVOKljgVLRmIEZbLYODAXT1RblGZlddwa4U-SmCkURBlVs6TC"
+        const val  CONTENT_TYPE = "application/json"
+
+    }
+}
```

Comment:
You could just have written an `object Constants`.
However using an `object` is overkill here. You can just write them as top-level definition without nesting them inside an object.